CMakeLists.txt

```diff
@@ -5,29 +5,27 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 add_definitions(-std=c++11 -Wall)
 # MSVC doesn't support -Wextra
-if(NOT WIN32)
+if(NOT MSVC)
     add_definitions(-Wextra)
-endif(NOT WIN32)
+endif()
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 add_executable(trojan
-	src/authenticator.cpp
-	src/clientsession.cpp
-	src/config.cpp
-	src/log.cpp
-	src/main.cpp
-	src/serversession.cpp
-	src/service.cpp
-	src/session.cpp
-	src/socks5address.cpp
-	src/ssldefaults.cpp
-	src/trojanrequest.cpp
-	src/udppacket.cpp
-	src/version.cpp)
+    src/authenticator.cpp
+    src/clientsession.cpp
+    src/config.cpp
+    src/log.cpp
+    src/main.cpp
+    src/serversession.cpp
+    src/service.cpp
+    src/session.cpp
+    src/socks5address.cpp
+    src/ssldefaults.cpp
+    src/trojanrequest.cpp
+    src/udppacket.cpp
+    src/version.cpp)
 find_package(Threads REQUIRED)
 target_link_libraries(trojan ${CMAKE_THREAD_LIBS_INIT})
-# COMPONENTS is used to generate linkage flags, e.g.: MSVC
-# It should match what we are using
-find_package(Boost 1.54.0 REQUIRED COMPONENTS system date_time)
```

Comment:
Why？It's required to generate msbuild project

Reply:
Fixed in https://github.com/trojan-gfw/trojan/pull/40/commits/4ecc2ac272f32c624720c1c135289804a770741f#diff-af3b638bc2a3e6c650974192a53c7291R31.